### ISSUE TITLE
Background color for the Message Icon on "All Messages Read" can't be changed

### ISSUE THREAD

**Issue by bauerp** [Thomas101/wmail#599]
* Which version of WMail are you using? 2.1.1

* Which Operating System are you using? Windows 7

When I change the background and/or transparency for the "All Messages Read" icon, nothing happens; however, it does work for the "Unread Messages".

**Comment by Thomas101** [comment]
Hi, it's not 100% clear but there's two sliders in the color picker. The bottom one is the transparency level which by default is set to complete transparency. Can you make sure this is all the way to the right?

**Comment by bauerp** [comment]
Yes, I’m aware of them and it changes nothing when slighting them to the
right. They work for the other icon though.

**Comment by Thomas101** [comment]
Hmmm, that's weird. If you change the colour and restart WMail does that make any difference?

**Comment by bauerp** [comment]
Didn’t do anything. Any changes made in the Background button of the left
icon have no effect at all. No big deal though, the default works fine for
me.

**Comment by Thomas101** [comment]
I've just stumbled across what this is. I'll take a look, thanks!
